### PR TITLE
add oclif manifest

### PR DESCRIPTION
**Pull request by jdxcode**
this makes the plugin compatible with the new oclif-based CLI

### CODE REVIEW

**Comment by RasPhilCo** [comment]
@uhoh-itsmaciek @jdxcode looks like we need to bump node in here. that ok?

**Comment by uhoh-itsmaciek** [comment]
@jdxcode btw, can you give us a heads up when you bump the CLI node version? It has broken us before (nothing serious, but better safe than sorry).

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "heroku-redis",
+  "name": "@heroku-cli/plugin-redis",
   "description": "Heroku plugin to manage Heroku Redis",
-  "version": "1.3.0",
+  "version": "1.3.2",
```
Comment (uhoh-itsmaciek) [inline package.json]: Can we do version bumps separately via `np`? I'm happy to do it myself or fill in anything that's missing in the docs.

Comment (jdxcode) [reply package.json]: I wanted to do that originally but I think on this PR I messed up the first couple of releases so I had to change the version number. 1.3.2 is already released, so if you wanted to release a new one you would have to release 1.3.3. We're happy to change it back to 1.3.0 though if that's what you would prefer

Comment (uhoh-itsmaciek) [reply package.json]: Ok that's fine then.

**Pull request merged**